unified_address.py

```diff
@@ -125,13 +50,15 @@ def randbytes(l):
 
         is_p2pkh = rand.bool()
         receivers = [
```

Comment:
I suggest representing this as a dict, which structurally enforces that keys are unique.

Reply:
The reason I didn’t represent it as a dict is that I wanted  data structure that gives a stable and predictable order; does a dict have that property? If not, the generated test vectors won’t be stable.

Reply:
`dict` became deterministic in Python 3.7, but you can also use [OrderedDict](https://docs.python.org/3/library/collections.html#ordereddict-objects).